cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/dataset_selector.spec.js

```diff
@@ -111,16 +111,15 @@ describe('dataset selector', { scrollBehavior: false }, () => {
     it('create index pattern and select it', function () {
       // Create and select index pattern for data_logs_small_time_1*
       cy.createWorkspaceIndexPatterns({
-        url: `${BASE_PATH}`,
-        workspaceName: `${WORKSPACE_NAME}`,
+        workspaceName: WORKSPACE_NAME,
```

Comment:
Thanks for correcting this. But should we make the workspaceName to be random as suggested by @AMoo-Miki? I am thinking using 
```
const randomString = Math.random().toString(36).substring(7);
const workspace = `${WORKSPACE_NAME}-${randomString}`;
```

Reply:
Oh I see you updated in cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/saved_search.spec.js

since it is a workspace name, do you think having a `-workspace-` is better? well your call. I am fine with a pure random string or a fixed string + some random.

Reply:
i'm fine with both! But since its going to be just used by the tests and (hopefully) removed by the end of the test, does it matter? Not that i think we'll ever encounter collision, but there is less of a chance of there being a collision when we allow for more random characters.

And yeah, both randomizers are confided in this file for least obstruction, but i can move it to a shared, although if we do so I think it will be safer to expose not as a constant but rather as a function

Reply:
we synced offline about this. For consistency and future expansion of test cases, i will add this change in